.gitignore

```diff
@@ -4,3 +4,25 @@ resources/
 node_modules/
 tech-doc-hugo
 
+### JetBrains template
```

Comment:
You may have left some .gitignore commits in your PR by mistake?

Reply:
It was intended. Following [Kubernetes](https://github.com/kubernetes/kubernetes) development practices